### ISSUE TITLE
Game is stuck after tricking Brax into leaving his office

### ISSUE THREAD

**Issue by zR1OQicz** [Innoxia/liliths-throne-public#1402]
**Describe the bug**
I am stuck after tricking Brax into leaving his office.
The Exit button and the dialog with Brax don't disappear after clicking Exit (but the mini-map has changed and shows me in front of the building).
I can't move or access my inventory.

As workaround, I can load the autosave that is generated after leaving the Enforcer HQ.

**Expected behavior**
I can continue the game after tricking Brax.

**To Reproduce**
1. Acquire the "Observant" Perk.
2. Follow the main quest until the meeting with Brax.
3. Use the option to trick Brax into leaving his office.
4. Click "Exit" to leave the building.

**Attach files**
error.log:
```
Version: 0.3.9.4
Java: 14.0.1
Warning: Nashorn engine is planned to be removed from a future JDK release
var timer not found...
Warning: Nashorn engine is planned to be removed from a future JDK release
```

**Additional information:**
- Game version v0.3.9.4
- OS: SuSE Linux 64-bit
- self-compiled version from github commit 1532da80b1980c40a0605d62ff6489afe9637f82
- java --version
  openjdk 14.0.1 2020-04-14
  OpenJDK Runtime Environment (build 14.0.1+7-suse-1.3-x8664)
  OpenJDK 64-Bit Server VM (build 14.0.1+7-suse-1.3-x8664, mixed mode)

**Comment by AceXP** [comment]
Thanks for the report, the cause of the bug is known and will likely be fixed in the next version.

**Comment by Ian-Luminus** [comment]
Just curious, does the fix apply to if you trick him, then beat him in combat as well? Because I ran into the same issue doing that set of choices.

**Comment by AceXP** [comment]
If I remember correctly, there were three or four situations where this could happen. They have all been fixed (by Inno).

**Comment by zR1OQicz** [comment]
Verified fixed in v0.3.9.8
